src/sign.ts

```diff
@@ -63,13 +63,11 @@ async function main(): Promise<void> {
     }
     case 'westend': {
       SS58_FORMAT = 42;
-      console.warn(`TODO: Need metadata for Westend. Likely to get BadProof error.`)
-      break;
+      throw Error(`TODO: Need metadata for Westend. Likely to get BadProof error.`);
```

Comment:
Same with the specName, when the metadata doesnt exist it should throw not warn.